examples/src/main/java/kafka/examples/ExactlyOnceMessageProcessor.java

```diff
@@ -152,17 +160,19 @@ public void run() {
                     consumer.seekToEnd(emptyList());
                     consumer.commitSync();
```

Comment:
Should we reset the `retries` to 0 here?

Reply:
Yep.